packages/api/src/axelar-query/isomorphic.ts

```diff
@@ -319,4 +345,144 @@ export class AxelarQueryAPIClient extends RestService {
 
     return this.cachedChainConfig;
   }
+
+  /**
+   * Get a list of active chains.
+   * @returns an array of active chains
+   */
+  public async getActiveChains(): Promise<string[]> {
+    if (!this.axelarQueryClient) {
+      this.axelarQueryClient =
+        await AxelarQueryClient.initOrGetAxelarQueryClient({
+          environment: this.env,
+          axelarRpcUrl: this.axelarRpcUrl,
+        });
+    }
+
+    return this.axelarQueryClient.nexus
+      .Chains({ status: ChainStatus.CHAIN_STATUS_ACTIVATED })
+      .then((resp) => resp.chains);
+  }
```

Comment:
i think we can remove the `AxelarQueryClient` and use [the existing cosmos package](https://github.com/axelarnetwork/axelarjs/tree/main/packages/cosmos) instead